global_config.json

```diff
@@ -3,6 +3,7 @@
   // "token": "<REPLACE ME>", // The auth token to access Answers experience.
   // "apiKey": "<REPLACE ME>", // The answers api key found on the experiences page. This will be provided automatically by the Yext CI system
   // "experienceVersion": "<REPLACE ME>", // the Answers Experience version to use for API requests. This will be provided automatically by the Yext CI system
+  // "environment": "<REPLACE ME>", // The environment to run on for this Answers Experience. (i.e. 'production' or 'sandbox')
```

Comment:
instead of `REPLACE ME` should the default value just be "production"?

Reply:
good point. updated!